EIPS/eip-223.md

```diff
@@ -0,0 +1,365 @@
+---
+eip: 223
+title: 223 Token with communication model
+description: Token with event handling and communication model
+author: Dexaran (@Dexaran)
+discussions-to: https://ethereum-magicians.org/t/erc-223-token-standard/12894
+status: Review
```

Comment:
```suggestion
status: Draft
```
New EIPs must start in draft. This is a new EIP.

Reply:
This thing exists since 2017 https://github.com/ethereum/eips/issues/223 but at that time the submission process was different.

Reply:
Yes. Nonetheless,  this is a new EIP with the current process.